macros/src/main/java/com/avast/cactus/Ignored.java

```diff
@@ -0,0 +1,4 @@
+package com.avast.cactus;
+
+public @interface Ignored {
```

Comment:
How about calling this `GpbIgnored` when all the other annotations start with Gpb prefix?